index.md

```diff
@@ -19,7 +19,7 @@ customers happy.
     </ul>
     <div class="card-bottom">
       <div class="card-button">
-        <button class="hxBtn hxBtn--primary"><a href="{{site.baseurl}}getting-started/design.html">Full Designer Guide</a></button>
+        <a class="hxBtn hxBtn--primary" href="{{site.baseurl}}getting-started/design.html">View Designer Guide</a>
```

Comment:
From code review: remove card-bottom buttons altogether; card min-height to 20rem